content/en/docs/next/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Quickstart
+weight: 900
+description: Get etcd up and running in less than 5 minutes!
+---
+
+## Step 1 - Install etcd
```

Comment:
Strange, the headings aren't showing up in the table of contents.

Reply:
Please replace the hyphen by a colon.

Reply:
My personal preference would be to simplifying the heading text to be:

    ## Install etcd

If you really want to see a step number, then just put the number:

    ## 1. Install etcd

But I'd vote for a heading w/o a number :)
It's your call.

Reply:
I think simpler, and without a number is better.